woocommerce-gutenberg-products-block.php

```diff
@@ -18,7 +18,7 @@
 
 defined( 'ABSPATH' ) || exit;
 
-$minimum_wp_version = '5.5';
+$minimum_wp_version = '5.8';
```

Comment:
Regarding my comment in the `has_core_dependencies` method where we pull metadata from a file, we could potentially include minium_wc_version in the same way as this `$minimum_wp_version`.

Reply:
I had initially thought of that, but if we do that we'd have to either pass it through to the Bootstrap (or maybe Package class) as an argument, we make it `$minimum_wp_version` a global, or we do the Woo core version check in a callback on `plugins_loaded` implementing the `use` keyword on the action callback to pass through the $minimum_wp_version. I didn't think of trying the last item until just now so I guess we could go with that to avoid the extra code (and potential performance hit) in the Bootstrap class.

Reply:
See my [latest comment here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4484#discussion_r677697085) (and the following performance checks I did). I think we're okay with sticking with the current implementation given:

- it retains one place to change minimum required version for Woo.
- the alternative will likely lead to more code churn/refactoring.